base/v0_6_exp/validate.go

```diff
@@ -29,10 +30,24 @@ func (rs Resource) Validate(c path.ContextPath) (r report.Report) {
 	if rs.Local != nil {
 		sources++
 		field = "local"
+		config := *rs.Local
+		_, report, err := exp.Parse([]byte(config))
+		if err != nil {
+			r.Merge(report)
+		} else {
+			r.AddOnError(c.Append("ignition", "config", "merge", "local"), common.ErrEmptyReport)
```

Comment:
So why is it an error when there is no error reported from the ign parse function?

Reply:
Do you mean if the report is empty then we error? if so the conditional logic here is wrong.